src/views/HomeView/StatsBar.vue

```diff
@@ -31,40 +31,42 @@
   </v-container>
 </template>
 
-<script>
-import { publishRest } from '@/rest';
+<script lang="ts">
+import { defineComponent, ref, computed } from '@vue/composition-api';
 import filesize from 'filesize';
+
+import { publishRest } from '@/rest';
 import SingleStat from '@/views/HomeView/SingleStat.vue';
 
-export default {
+export default defineComponent({
   name: 'StatsBar',
   components: { SingleStat },
-  data() {
+  setup() {
+    const dandisets = ref(0);
+    const users = ref(0);
+    const size = ref(0);
+
+    const stats = computed(() => [
+      {
+        name: 'dandisets',
+        value: dandisets.value,
+        description: 'A DANDI dataset including files and dataset-level metadata',
+        href: '/#/dandiset',
+      },
+      { name: 'users', value: users.value },
+      { name: 'total data size', value: filesize(size.value, { round: 0, base: 10, standard: 'iec' }) },
+    ]);
+
+    // equivalent of async created method in options API
+    publishRest.stats().then((data) => {
+      dandisets.value = data.dandiset_count;
+      users.value = data.user_count;
+      size.value = data.size;
+    });
```

Comment:
Do you prefer the promise chaining syntax? Personally I generally prefer `await`, although I admit it's pretty ugly in this case:
```
const { dandiset_count, user_count, size } = await publishRest.stats();
dandisets.value = dandiset_count;
users.value = user_count;
size.value = size;
```

Reply:
Yeah I generally prefer `async/await` too. In this case I used `.then()` because I think it makes it more obvious that those 4 lines are performing a single operation together, and separates them from the rest of the `setup()` boilerplate.